docs/deployment_groups.md

```diff
@@ -31,6 +31,14 @@ Creating a deployment group with the same name and labels as an existing one wil
 "not modified" response. Creating a deployment group with the same name but different labels as an
 existing one will return an error stating that the deployment group already exists.
 
+To figure out which labels are available and what sort of hosts each label key
+and value would return, you can use
```

Comment:
Can you change to just "you can use `helios hosts`"`? We'll assume the user by this point knows how the `hosts` subcommand works.